utils/src/main/kotlin/CommandLineTool.kt

```diff
@@ -92,15 +102,12 @@ interface CommandLineTool {
     /**
      * Get the version of the command by parsing its output.
      */
-    fun getVersion(
-        versionArguments: String = "--version", workingDir: File? = null,
-        transform: (String) -> String = { it }
-    ): String {
-        val version = run(workingDir, *versionArguments.split(' ').toTypedArray())
+    fun getVersion(workingDir: File? = null): String {
+        val version = run(workingDir, *getVersionArguments().split(' ').toTypedArray())
```

Comment:
It would make sense to switch the names of `version` and `versionString` because this variable here contains the raw version string returned from the tool, and the one below contains the processed version.

Reply:
I don't think switching names makes it any clearer as both are actually version strings. But how about `rawVersion` or `versionOutput`, and `transformedVersion`? And while at it, should the latter better be `processedVersion` and the function be called `processVersion`?

Reply:
I decided to stick with the current naming for now to keep the diff a bit smaller and get this merged quicker hopefully.